src/app/Components/sign-in/sign-in.component.html

```diff
@@ -23,6 +23,9 @@ <h1>Sign In</h1>
         Login
       </button>
     </div>
+    <div *ngIf="errorMessage">
+      <p class="errorMessageContainer">{{ errorMessage }}</p>
```

Comment:
is it necessary to have both a div and a p? suggestion: `<p *ngIf="errorMessage" class="errorMessageContainer">{{ errorMessage }}</p>`